components/director/internal/open_resource_discovery/validation_test.go

```diff
@@ -1493,6 +1519,86 @@ func TestDocuments_ValidateAPI(t *testing.T) {
 
 				return []*open_resource_discovery.Document{doc}
 			},
+		}, {
+			Name: "Not incremented `version` field when resource definition's URL has changed for API",
```

Comment:
All API/Event tests change the fields in the `resourceDefinitions` field. 

Probably, for completeness sake, add one more test case where some other field is changed and the version is not bumped.